PackageExplorer/Models/NuGetFrameworkModel.cs

```diff
@@ -0,0 +1,123 @@
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = ".NET Framework 4 and higher", ShortName = "net4", Framework = Frameworks.FindByIdentifierShortName("net40").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = ".NET Framework 4.0.3 and higher", ShortName = "net403", Framework = Frameworks.FindByIdentifierShortName("net403").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = ".NET Framework 4.5 and higher", ShortName = "net45", IsDefault = true, Framework = Frameworks.FindByIdentifierShortName("net45").FirstOrDefault() });
+			frameWork.SelectedItem = frameWork.Items.FirstOrDefault(x => x.IsDefault);
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Silverlight" };
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Silverlight 4 and higher", ShortName = "sl4", Framework = Frameworks.FindByIdentifierShortName("sl4").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Silverlight 5", ShortName = "sl5", IsDefault = true, Framework = Frameworks.FindByIdentifierShortName("sl5").FirstOrDefault() });
+			frameWork.SelectedItem = frameWork.Items.FirstOrDefault(x => x.IsDefault);
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Windows Phone" };
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Windows Phone 7 and higher", ShortName = "wp7", Framework = Frameworks.FindByIdentifierShortName("wp7").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Windows Phone 7.5 and higher", ShortName = "wp71", Framework = Frameworks.FindByIdentifierShortName("wp75").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Windows Phone 8 and higher", ShortName = "wp8", Framework = Frameworks.FindByIdentifierShortName("wp8").FirstOrDefault() });
+			frameWork.Items.Add(new NuGetTargetFrameworkItem { Name = "Windows Phone 8.1 and higher", ShortName = "wpa81", IsDefault = true, Framework = Frameworks.FindByIdentifierShortName("wpa81").FirstOrDefault() });
+			frameWork.SelectedItem = frameWork.Items.FirstOrDefault(x => x.IsDefault);
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = ".NET for Windows Store apps", ShortName = "win8", Framework = Frameworks.FindByIdentifierShortName("win8").FirstOrDefault() };
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Xamarin For iOS", ShortName = "xamarinios", Framework = Frameworks.FindByIdentifierShortName("xamarinios").FirstOrDefault() };
+			PortableFrameworks.Add(frameWork);
+
+			frameWork = new NuGetTargetFrameworkItem { Name = "Xamarin For Android", ShortName = "MonoAndroid", Framework = Frameworks.FindByIdentifierShortName("MonoAndroid").FirstOrDefault() };
+			PortableFrameworks.Add(frameWork);
+		}
```

Comment:
I am not sure I understand, there is nothing in here that is duplicate.  This is the core of the most recent enhancement, it moves ALL of the target platforms into a single are and leverages the NuGet libraries. Everything in here is needed to support the different platforms within the selection dialog.  Eventually I'd love to move this into the NuGet libraries itself (or at least something similar to this code).